src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -728,6 +728,16 @@ def collect_periscope_logs(resource_group_name, name, storage_account_name=None,
     if kube_context:
         kubectl_prior.extend(["--context", kube_context])
 
+    fd, temp_yaml_path = tempfile.mkstemp()
+    temp_yaml_file = os.fdopen(fd, 'w+t')
+    deployment_yaml = urlopen(
+        "https://raw.githubusercontent.com/Azure/aks-periscope/latest/deployment/aks-periscope.yaml").read().decode()
+    
+    if not storage_account_name or not sas_token:
+        print('No storage account specified. Downloading logs to local machine.')
+        apply_periscope_yaml(kubectl_prior, deployment_yaml, temp_yaml_file, temp_yaml_path)
+        copy_and_zip_periscope_files(kubectl_prior)
```

Comment:
We are not exporting file locally, if storage account details are provided correctly?

Reply:
Yes that's correct.